R/pkg/R/functions.R

```diff
@@ -3048,6 +3048,26 @@ setMethod("array_position",
             column(jc)
           })
 
+#' @details
+#' \code{array_repeat}: Creates an array containing the left argument repeated the number of times
+#' given by the right argument.
+#'
+#' @param count Column or constant determining the number of repetitions.
```

Comment:
change to `@param count a Column or constant determining the number of repetitions.`